json/Side_Text.txt

```diff
@@ -228828,1335 +228828,1335 @@
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "任務なんだろ？<%br>はやく行こうぜ！",
-		"tr_text": "",
+		"tr_text": "We've got a mission, yeah?<%br>Let's get movin'!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "13",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "（じょうほうを、しゅうせきして持ちかえる。<%br>　それがオレが作られた理由だからって……）",
-		"tr_text": "",
+		"tr_text": "(This is the reason I was created...<%br>To gather and report information.)",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "14",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "（ふたりに言ったら、どうなるんだろう？）",
-		"tr_text": "",
+		"tr_text": "(What would the others do if I told them that?)",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "はぁ……はぁ……はぁ……！",
-		"tr_text": "",
+		"tr_text": "＊pant＊... ＊pant＊... ＊pant＊...!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "ジェネ、だいじょうぶか？",
-		"tr_text": "",
+		"tr_text": "Gene, you OK?",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "……はい、大丈夫です！<%br>モアさっきは……えっと、いつもありがとう。",
-		"tr_text": "",
+		"tr_text": "...I'm fine, I'm fine!<%br>More, about earlier... I wanted to thank you.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "え……？",
-		"tr_text": "",
+		"tr_text": "Wassat...?",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "さっき、わたしのこと庇ってくれたでしょ？",
-		"tr_text": "",
+		"tr_text": "You were trying to protect me before, weren't you?",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "それに……<%br>いつも優しくしてくれて、ありがとう。",
-		"tr_text": "",
+		"tr_text": "So...<%br>Thank you for always being so nice to me.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "……べ、べっつに、ふつうだろ！",
-		"tr_text": "",
+		"tr_text": "...C-come on, that's nothin' special!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "……モアと、チームが組めてよかった。<%br>仲間になれて、よかった。",
-		"tr_text": "",
+		"tr_text": "...It was fun teaming up with you, More.<%br>I'm glad we're friends.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "……なか……ま……",
-		"tr_text": "",
+		"tr_text": "...We're... friends...",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "そ、そうだ……！<%br>仲間なんだから、とーぜんだろっ！",
-		"tr_text": "",
+		"tr_text": "Y-yeah, you got it!<%br>What are friends for, right?!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "ジェネは、抜けてるしたよりないし……<%br>それに、ちょっと変わってると思う。",
-		"tr_text": "",
+		"tr_text": "Gene's still messin' up all over the place...<%br>And I gotta be honest, she's a real weirdo.",
```

Comment:
Gene's clumsy, but More uses the word `incompetent` and `eccentric` to describe her.

Reply:
How's this?

Reply:
Should now be good.